json_structure.puml

```diff
@@ -0,0 +1,257 @@
+@startuml
```

Comment:
what is this language? can we have some background? is this used?

Reply:
this is only a visualisation. you can use it in your IDE or for example at plantuml.com.
The json_structure is like this:
http://www.plantuml.com/plantuml/svg/fLV1ZkCs3BtdAuoSMXJR3mYAmGABja2nYosm2_IiM8p3ZYmP53tZTD1_B-Guh_AC9qASKqnQUPJ8noVwOrGZshTk4qahRrgAdQdf6tL6Z0QfThlDfdOcncghxCZkTkFFpzQ8xdPVJFT737vRlMogwgTlAkoRQc_gnqP2xsqTN93qXhrMTMWxeHWvU4TFv9B_LyiQ998g-oOXLPHxukM_H-BcgB1eY6p9aM_qc9px45p5_i2UbPBZCqVzvRCvaIJGXygbHw1v6dJqiCrTOVytrSesEqeG9gz6ENZ0_YcX8z7J6RaxMqmn-vkGjoH1B4dcZMgQXiGC7DzwsLiQClUOvIUIbAvfawCFcJXlvZixYgJdlUZPYFdwPxPwp75F7INJzstr2XvBlvYkqf0mV_Xny-zcSxNouSE7HMgyAUrs7GwgkFB_ADTwY2p9MtH3-SZeYq_7tZ-ExAxJK-H_Q2J_q0NHqK_ZOpdtot6kR1sXj_SYORV8GI7WJ1EiKRFSp61TCeRBmsaf5uXwPNFpi8SWhT6S_aOQqfl0mS24ZSR3EH-cecP1CgN3UlRwSCa2aYBc-RL8aA-3PTyi9sicJjoQvghQOMNITIpyEihSUMKOdyUbDlHxHrKu72BfyCPpoZsTgLbF9KM43ertp32pJZEfyugRuk8JusDdXF1gfc0G_IUEl7UqWao5IfSRQYK2bgn7G0NNGynL4Uo_7gnwGTx9EUuYyjwmpVYnp5am4I1TOA__ioXP4uM2EpN1l_VAiQSY-RBzIC0L3X9HspcZ8ynt6iugh3HeaMX01TScE--PCkFcsRrQAvn-kne2IReB9IMgS05Wjw1B6ZI6wU5MMKV0Z7dxu792GHsW-5uFfcMNAypHE6uy85YDunhc8udF5BVZWLowex4Wgl0ZXNv63XeK2tflISwin8AdUSndCTrjXqVDa81v4yruEW_SqosXCE3SCCgxllNGzmDBr4a2VXUsOpCridnPiTVHm2IRkMRotHffsBzxxEdk6hFSajTRvlCOUgcnc_J5a-gFcPf43FjffZuRivcw3htNcsRrbE2lXyEKu_5bIEPyaC-c-6iZVFbkpATxHFZB-ERRvLFXsVWcaiQoJkKCAUZ3fVRRzH1Peku30WlkWm4NxZxLdH2eNa5-KOFgPJv2Bd_L5hxz4AbOAiHRFyJ_9gQbLrYs5wElsehjRRYeuRAEbcK3aHVY5WEM25giA2gFxQETFwNsdpv2WdYIKG_YUsl8DeGLToK1w2EpeIYGhdAISAmYwt60xMeS9FjE69ouN3Fi6rLR6bYGVqoEdOIzsRDZFHMOLmYFVMQpapquxc0TBnQ2ODYzlC6dwrZUMnilsmB0qEGFbV7SpfhxZe41dppJM926bkxzj9R8ZeDZ07XroTvoRzxrELW6G4RN0ioRuoDvsxVkFm00

Reply:
I dont think we need this in the repo. Most IDE's already allow you to generate an overview of your class structure. And otherwise you can easily generate it with any tools.

Now we also have to keep this updated whenever we update our classes. While our classes already define a structure. So sounds to me as double work

Reply:
ok, i remove them